python/recursion/factorial.py

```diff
@@ -0,0 +1,11 @@
+def factorial(n: int) -> int:  # Python >= 3.5
+    if n == 0:
+        return 1
+    if n < 0:
```

Comment:
Factorial for negative numbers is not defined. 
Can you just print a message saying that instead of computing it? I think that will be better. :)

Reply:
Okay, I'll do it but after 2 hours or so [busy atm]

Reply:
Sure. There is another PR with the same program in python as well. Can you create another PR for some other basic program as well? Anything is fine by me.

Reply:
I made it raise a ValueError, then added a try/except block in the main function

Reply:
Perfect!